ocean_lib/example_config.py

```diff
@@ -5,78 +5,113 @@
 
 import logging
 import os
-from typing import Dict, Optional
+import copy
+
+from web3 import Web3
+
+from ocean_lib.config import (
+    Config,
+    SECTION_ETH_NETWORK,
+    NAME_CHAIN_ID,
+    SECTION_RESOURCES,
+    NAME_NETWORK_URL,
+    NAME_PROVIDER_URL,
+    config_defaults,
```

Comment:
can you please keep the constants in one part, and Config and config_defaults either at the beginning of these imports or at the end?

Reply:
Sure